src/controller.cpp

```diff
@@ -575,10 +575,10 @@ geometry_msgs::Twist Controller::update(const double target_x_vel,
 
   /***** Compute forward velocity *****/
   // Apply acceleration limits and end velocity
-  double current_target_acc;
-  double acc;
-  double t_end_phase_current;
-  double d_end_phase;
+  double current_target_acc = 0;
+  double acc = 0;
+  double t_end_phase_current = 0;
+  double d_end_phase = 0;
```

Comment:
It might make sense to declare (and maybe assign) `t_end_phase_current` and `d_end_phase` here, because they are assigned their values in a smaller scope in the next block of code. But the `current_target_acc` and `acc` variables are assigned their values much later in the same scope, so why not move their declarations there?